src/styles/filter.styles.js

```diff
@@ -0,0 +1,14 @@
+export const filtersStyles = {
+  position: 'relative'
+};
+
+export const filterDropdownStyles = {
+  position: 'absolute',
+  maxHeight: '300px',
+  overflow: 'overlay'
```

Comment:
Maybe we should keep it to `hidden`, and let user manage scroll inside of their component.